### ISSUE TITLE
Highlighted element is not visible, solid background covers the content

### ISSUE THREAD

**Issue by jackie-d** [kamranahmedse/driver.js#241]
I'm experimenting an issue with this JS tool.

I included the css and js from the dist folder.

I configured as follow:

```
$(document).ready(() => {

    const driver = new Driver({
        animate: true,
        opacity: 0.7,
        padding: 10,
        stageBackground: '#ffffff',
        showButtons: true,
        keyboardControl: true,
        allowClose: false
    });

    driver.defineSteps([
            {
                element: '#element > a > span',
                popover: {
                    title: 'Title',
                    description: 'Description',
                    position: 'right-center'
                }
            }, {
                element: '#element> div > span.my-class',
                popover: {
                    title: 'Title',
                    description: 'Description',
                    position: 'right-center'
               }
           }
        ]
    );

    driver.start();
});
```

I'm getting the result:

![Schermata 2020-05-26 alle 15 36 46](https://user-images.githubusercontent.com/20744688/82907557-0d313d00-9f67-11ea-8c7c-0dc709f0c310.png)

As you can see the highlighted content is totally covered by the white background, and in the DOM the highlighted element is empty.

The same beheviour presents on any element of the page, and with any stageBackground solid color.

Is there a way to configure it properly to get a result as in the examples?

Thanks.

**Comment by kamranahmedse** [comment]
It seems to be an issue with positioning. Can you please try with `animate: false` and see if it fixes the issue. Also, please provide a codesandbox example so that we can help better.

**Comment by jackie-d** [comment]
Yes, I confirm the issue is solved with `animate: false`

**Comment by PeterPan73** [comment]
Hi
@kamranahmedse When will fix problem with modals in Bootstrap?
Thanks.

**Comment by jenslw** [comment]
> It seems to be an issue with positioning. Can you please try with `animate: false` and see if it fixes the issue. Also, please provide a codesandbox example so that we can help better.

@kamranahmedse 
I made a quick codepen demonstrating the problem: https://codepen.io/jenslw/pen/qBZrOyE
Do you have any idea how to solve this?

**Comment by de1ck** [comment]
> Yes, I confirm the issue is solved with `animate: false`

useful  thank you

**Comment by mewalig** [comment]
am seeing similar issues w bootstrap menu items, with or without animate

**Comment by aldoadhi** [comment]
Quick fix using CSS, put this after driver's CSS:

`
#driver-highlighted-element-stage {
    background: transparent!important;
    outline: 5000px solid rgba(0, 0, 0, .75);
}

#driver-page-overlay {
    background: none !important;
}
`

**Comment by pagarevijayy** [comment]
> Quick fix using CSS, put this after driver's CSS:
> 
> ```
> #driver-highlighted-element-stage {
>     background: transparent!important;
>     outline: 5000px solid rgba(0, 0, 0, .75);
> }
> 
> #driver-page-overlay {
>     background: none !important;
> }
> ```

This hack worked, thanks @aldoadhi

**Comment by mukesh-inoid** [comment]
still I have this issue with materialize side nav bar.  please anyone can fix this. Thank you.
![Screenshot 2023-01-19 130040](https://user-images.githubusercontent.com/107191507/213392894-e115808c-e690-46b0-9db0-cceca82db27c.png)

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. This issue should no longer exist. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
